system-variables: update noop functions (#3898)
pingcap/docs: system-variables.md

- Scope: SESSION | GLOBAL
- Default value: 0

> - By default, TiDB returns an error when attempting to use syntax for functionality that is not yet implemented. However, some users may prefer that TiDB silently ignore cases when the functionality is not available. This is helpful in cases where you can not make changes to the SQL code.

Reviewer: ```suggestion
- By default, TiDB returns an error when you attempt to use the syntax for functionality that is not yet implemented. However, you might also prefer that TiDB silently ignores such cases of unavailable functionality, which is helpful if you cannot make changes to the SQL code.
```

- Scope: SESSION | GLOBAL
- Default value: 0

> - By default, TiDB returns an error when you attempt to use the syntax for functionality that is not yet implemented. When set to `1`, TiDB silently ignores such cases of unavailable functionality, which is helpful if you cannot make changes to the SQL code.

Reviewer: ```suggestion
- By default, TiDB returns an error when you attempt to use the syntax for functionality that is not yet implemented. When the variable value is set to `1`, TiDB silently ignores such cases of unavailable functionality, which is helpful if you cannot make changes to the SQL code.
```
